test/cmd/basicresources.sh

```diff
@@ -65,6 +65,10 @@ echo "expose: ok"
 
 oc delete all --all
 
+# switch to test user to be sure that default project admin policy works properly
+oc policy add-role-to-user admin test-user
+oc login --server=${KUBERNETES_MASTER} --certificate-authority="${MASTER_CONFIG_DIR}/ca.crt" -u test-user -p anything
```

Comment:
> any way to avoid requiring these envvars to be set? the other test cmd scripts can be invoked against a running server directly, which is nice

done